submissions/zhenyakornilov/frogger-game/app.js

```diff
@@ -0,0 +1,141 @@
+const EDGE_POSITION = {
+  rightX: 404,
+  leftX: 0,
+  topY: -15,
+  bottomY: 390,
+};
+
+const PLAYER_START_POSITION = {
+  x: 202,
+  y: 390,
+};
+
+const MOVE_STEP = {
+  axisX: 101,
+  axisY: 81,
+};
+
+const GAME_BOARD_WIDTH = 505;
+
+class Character {
+  constructor(x, y, sprite) {
+    this.x = x;
+    this.y = y;
+    this.sprite = sprite;
+  }
+
+  render() {
+    ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+  }
+}
+
+class Enemy extends Character {
+  constructor(x, y, speed, direction, sprite, player) {
+    super(x, y, sprite);
+    this.speed = speed;
+    this.direction = direction;
+    this.player = player;
+  }
+
+  update(dt) {
+    if (this.direction === "left") {
+      this.x -= dt * this.speed;
+      if (this.x < 0) {
+        this.x = GAME_BOARD_WIDTH + 10;
```

Comment:
WHat is `10` here and across the codebase? I guess `-10` is the same but just negative. If true then having `OFFSET = 10` and referring to it as `OFFSET` and `-OFFSET` would be fine.
Of course `OFFSET` word doesn't explain it application, so invent a better name.